File: simple-demo/go/simple-demo.go
// Kledo API Demo - Go (tanpa dependency eksternal)
// Jalankan: go run simple-demo.go

package main

import (
	"encoding/json"
	"fmt"
	"net/http"
	"os"
)

func main() {
	apiHost := "http://xxx.api.kledo.com/api/v1"
	accessToken := "your_token_here"

	req, _ := http.NewRequest("GET", apiHost+"/finance/accounts?sort_by=name&order_by=asc&per_page=5", nil)
	req.Header.Set("Accept", "application/json")
	req.Header.Set("Authorization", "Bearer "+accessToken)

	res, err := http.DefaultClient.Do(req)
	if err != nil {
		fmt.Fprintln(os.Stderr, err)
		return
	}
	defer res.Body.Close()

	var data interface{}
	json.NewDecoder(res.Body).Decode(&data)
	out, _ := json.MarshalIndent(data, "", "  ")
	fmt.Println(string(out))
}

File: simple-demo/ruby/simple-demo.rb
# Kledo API Demo - Ruby (tanpa gem eksternal)
# Jalankan: ruby simple-demo.rb

require 'net/http'
require 'json'

API_HOST = 'http://xxx.api.kledo.com/api/v1'
ACCESS_TOKEN = 'your_token_here'

uri = URI("#{API_HOST}/finance/accounts?sort_by=name&order_by=asc&per_page=5")
req = Net::HTTP::Get.new(uri)
req['Accept'] = 'application/json'
req['Authorization'] = "Bearer #{ACCESS_TOKEN}"

res = Net::HTTP.start(uri.hostname, uri.port) { |http| http.request(req) }
data = JSON.parse(res.body)
puts JSON.pretty_generate(data)
